ui/public/i18n/language.json

```
@@ -319,7 +319,9 @@
     "invalid_backup_format": "Backup not found or invalid",
     "remap_at_least_one_green": "You have to select at least one green to remap",
     "assign_later": "Assign later",
-    "current_int": "System interfaces"
+    "current_int": "System interfaces",
+    "restore_network": "Restore network configuration",
+    "skip_network_info": "Current network configuration will be preserved, but some applications may be disabled at the end of the process. After restore, review the system configuration to re-enable all required applications."
```

Is it better with gerund?

```suggestion
    "skip_network_info": "Current network configuration will be preserved, but some applications may be disabled at the end of the process. After restoring, review the system configuration to re-enable all required applications."
```